scripts/cluster/create_multinode_cluster.sh

```diff
@@ -42,6 +42,11 @@ mkdir -p $HOME/.kube
 sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
 sudo chown $(id -u):$(id -g) $HOME/.kube/config
 
+# enabling PodNodeSelector for control plane node affinity
```

Comment:
reason for this change?

Reply:
The old method of trying to fix this. Will be deleted.